### PR TITLE
Add drupal.

### PR DESCRIPTION
**Pull request by steveiliop56**
In this pr I add drupal. (dont think I need to explain what drupal is : ) )

### CODE REVIEW

File: apps/drupal/config.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "$schema": "../schema.json",
+  "name": "Drupal",
+  "port": 8215,
+  "available": true,
+  "exposable": true,
+  "id": "drupal",
+  "tipi_version": 1,
+  "version": "php8.2-fpm-bookworm",
+  "categories": [
+    "social"
```
Comment (meienberger) [inline apps/drupal/config.json:11]: I don't think this category fits well drupal

Comment (steveiliop56) [reply apps/drupal/config.json:11]: Haven has this that's why I added this...

Comment (meienberger) [reply apps/drupal/config.json:11]: Haven is a social network for your family so I think it fits. I would use "development"

**Comment by steveiliop56** [comment]
It has a gui

File: apps/drupal/config.json (hunk 1 above)
Comment (steveiliop56) [reply apps/drupal/config.json:11]: Ok using that

**Pull request closed without merging**